src/libraries/Microsoft.Extensions.Caching.Memory/src/CacheEntry.CacheEntryTokens.cs

```diff
@@ -35,7 +36,7 @@ internal void AttachTokens(CacheEntry cacheEntry)
                             if (expirationToken.ActiveChangeCallbacks)
                             {
                                 _expirationTokenRegistrations ??= new List<IDisposable>(1);
-                                IDisposable registration = expirationToken.RegisterChangeCallback(ExpirationCallback, cacheEntry);
+                                IDisposable registration = expirationToken.RegisterChangeCallback((Action<object?>)ExpirationCallback, cacheEntry);
```

Comment:
Why's this necessary?

Reply:
`ExpirationCallback` is `Action<object>`, and there's a warning if I don't cast it explicitly